api/models.py

```diff
@@ -2,18 +2,20 @@
 from django.db import models
 from django.contrib.auth.models import AbstractUser
 from django.contrib.postgres.fields import JSONField
+from typedmodels.models import TypedModel
 
 
 class User(AbstractUser):
     id = models.AutoField(primary_key=True)
     gravatar = models.URLField(blank=True)
 
 
-class Collection(models.Model):
+class CollectionBase(TypedModel):
     title = models.TextField()
     description = models.TextField(null=True, blank=True)
     tags = models.TextField(null=True, blank=True)
     created_by = models.ForeignKey(User)
+    created_by_org = models.TextField(null=True, blank=True)
```

Comment:
It looks like `CharField` require specifying a max_length while `TextField` does not. Is that preferred over `TextField` if the input is not expected to be very large?